docs/develop.md

```diff
@@ -0,0 +1,239 @@
+# Development
+
+There is many way to intall a development environement for uData.
+The prefered one is to use virtualenv and Docker/Docker-Compose.
+
+## System dependencies
+
+The uData development environment requires the following system dependencies
+(on an Debian/APT based system):
+
+* build-essential
+* pkg-config
+* python-dev
+* python-virtualenv
+* libjpeg-dev
+* zlib1g-dev
+* libpng12-dev
+* libtiff5-dev
+* libfreetype6-dev
+* liblcms2-dev
+* libopenjpeg-dev
+* libwebp-dev
+* libpng12-dev
+* libxml2-dev
+* libxslt1-dev
+* liblzma-dev
+
+```shell
+apt-get install build-essential pkg-config python-dev python-virtualenv libjpeg-dev zlib1g-dev libpng12-dev libtiff5-dev libfreetype6-dev liblcms2-dev libopenjpeg-dev libwebp-dev libpng12-dev libxml2-dev libxslt1-dev liblzma-dev
+```
+
+## Python dependencies
+
+It is recommanded to work within a virtualenv to ensure proper dependencies isolation.
+
+It is recommanded to have a parent workspace where project are cloned, data stored...
+It will be designated by `$WORKSPACE` until the end of the documentation.
+
+```shell
+$ mkdir $WORKSPACE && cd $WORKSPACE
+```
+
+First, create and activate your virtualenv, with one of the following command
+(depending on you using virtualenv or virtualenvwrapper):
+
+```shell
+$ virtualenv . && source bin/activate  # for virtualenv
+$ mkvirtualenv udata && setvirtualenvproject .  # for virtualenvwrapper
+$ mkproject udata  # for virtualenvwrapper with configured project dir
+```
+
+If your system is using Python 3 as primary Python installation,
+don't forget to specify the python version with the `-p` parameter:
```

Comment:
_specify the Python 2 version_? As you're talking about Python 3 a few words earlier it may be a good idea to remind the expected version and avoid confusion.